src/lib/y2network/interface_config_builders/bridge.rb

```diff
@@ -0,0 +1,40 @@
+require "yast"
+require "y2network/interface_config_builder"
+
+Yast.import "LanItems"
+
+module Y2Network
+  module InterfaceConfigBuilders
+    class Bridge < InterfaceConfigBuilder
+      def initialize
+        super(type: "bridge")
+      end
+
+      def bridgable_interfaces
+        Yast::LanItems.GetBridgeableInterfaces(name)
+      end
+
+      def already_configured?(devices)
+        configurations = Yast::NetworkInterfaces.FilterDevices("netcard")
+        netcard_types = (Yast::NetworkInterfaces.CardRegex["netcard"] || "").split("|")
+
+        confs = netcard_types.reduce([]) do |res, devtype|
+          res.concat((configurations[devtype] || {}).keys)
+        end
+
+        devices.each do |device|
```

Comment:
```NetworkInterfaces::List("netcard")``` and 8 lines above can be dropped

Reply:
any reason why? I think it is needed to detect if device is already configured, not?